documentation/projects/proposals/popularity_optimizations/20230420-implementation_plan_popularity_optimizations.md

```diff
@@ -0,0 +1,486 @@
+# Implementation Plan: Decoupling Popularity Calculations from the Data Refresh
+
+<!-- See the implementation plan guide for more information: https://github.com/WordPress/openverse/tree/19791f51c063d0979112f4b9f4eeace04c8cf5ff/docs/projects#implementation-plans-status-in-rfc -->
+<!-- This template is exhaustive and may include sections which aren't relevant to your project. Feel free to remove any sections which would not be useful to have. -->
+
+## Reviewers
+
+<!-- Choose two people at your discretion who make sense to review this based on their existing expertise. Check in to make sure folks aren't currently reviewing more than one other proposal or RFC. -->
+
+- [ ] @AetherUnbound
+- [ ] @obulat
+
+## Project links
+
+<!-- Enumerate any references to other documents/pages, including milestones and other plans -->
+
+- [Project Thread](https://github.com/WordPress/openverse/issues/433)
+- [Project Proposal](https://docs.openverse.org/projects/proposals/popularity_optimizations/20230406-project_proposal_popularity_optimizations.md)
+
+## Overview
+
+<!-- A brief one or two sentence overview of the implementation being described. -->
+
+This document describes a method for separating the popularity calculation steps
+from the data refresh process, allowing the data refresh to be re-enabled
+independently of popularity calculations.
+
+## Background
+
+To improve search relevancy, we boost records in our search results relative to
+a normalized popularity score. Because the process of calculating these scores
```

Comment:
Doesn't this have a different meaning? "relative to" seems precise in that individual document popularity scores are "relative" to the normalised. It's not clear what "using" would mean here. Using _how_, I suppose would be the clarification I would need, but "relative" already implies it, assuming a general understanding of statistics (weighted/relative scores are common enough, I think?).

Not trying to be picky, I just want to make sure that if the wording here changes that "using" is disambiguated because it isn't obvious on its own in a way that I do think "relative" to is, at least somewhat.

Reply:
That's what I was aiming for with the wording, but I do also see how it could be confusing, especially without getting too deep into explaining the configuration. I tried looking up how the Elasticsearch docs phrase this. Maybe the easiest thing to do is to borrow their phrasing and link directly to the elasticsearch docs, something like:

```To improve search relevancy, we use a [rank feature query](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-rank-feature-query.html) to calculate relevance scores based (in part) on a `standardized_popularity` score.```

Reply:
That's far more precise :100: